pax-utils.yaml

```diff
@@ -27,7 +27,7 @@ pipeline:
   - uses: fetch
     with:
       uri: https://gitweb.gentoo.org/proj/pax-utils.git/snapshot/pax-utils-${{package.version}}.tar.gz
-      expected-sha256: 8baed2f9c5ae8e0cda1b9c75990864101afc64fad0a4616e10f3ff8ef891040b
+      expected-sha256: 179f35678a0472e32315fc8725c62a6be9549f8765178b26486851c4bbe54590
```

Comment:
Not sure why the sha is different if we are fetching the same tar ball just using a different location.  Is this something that needs to be looked at?